frontends/web/src/routes/account/add/components/coin-dropdown.module.css

```diff
@@ -0,0 +1,91 @@
+.dropdown {
+    appearance: none;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' fill='none' stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-labelledby='chevron down' color='%23777'%3E%3Cdefs/%3E%3Cpolyline points='6 10 12 16 18 10'/%3E%3C/svg%3E%0A");
```

Comment:
We use this inline svg in 4 places (coin-dropdown.tsx, countryselect.tsx, activecurrencies.tsx and singledropdown.tsx), each seem to define their own DropdownIndicator component. It looks you could re-use some components. WDYT?